vendor/modules.txt

```diff
@@ -102,41 +107,53 @@ github.com/hashicorp/hcl/hcl/token
 github.com/hashicorp/hcl/json/parser
 github.com/hashicorp/hcl/json/scanner
 github.com/hashicorp/hcl/json/token
-# github.com/hokaccha/go-prettyjson v0.0.0-20180920040306-f579f869bbfe
+# github.com/hokaccha/go-prettyjson v0.0.0-20190818114111-108c894c2c0e
 github.com/hokaccha/go-prettyjson
 # github.com/inconshreveable/mousetrap v1.0.0
 github.com/inconshreveable/mousetrap
-# github.com/influxdata/influxdb v1.6.4
+# github.com/influxdata/influxdb v1.7.9
 github.com/influxdata/influxdb/client/v2
 github.com/influxdata/influxdb/models
 github.com/influxdata/influxdb/pkg/escape
 # github.com/jmoiron/sqlx v1.2.1-0.20190319043955-cdf62fdf55f6
 github.com/jmoiron/sqlx
 github.com/jmoiron/sqlx/reflectx
-# github.com/konsorten/go-windows-terminal-sequences v1.0.1
+# github.com/konsorten/go-windows-terminal-sequences v1.0.2
 github.com/konsorten/go-windows-terminal-sequences
 # github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
 github.com/kr/logfmt
-# github.com/lib/pq v1.0.0
+# github.com/lib/pq v1.3.0
 github.com/lib/pq
 github.com/lib/pq/oid
+github.com/lib/pq/scram
 # github.com/magiconair/properties v1.8.1
 github.com/magiconair/properties
+# github.com/mainflux/mproxy v0.0.0-20200218164159-5d4b72740bc2
+github.com/mainflux/mproxy/pkg/events
+github.com/mainflux/mproxy/pkg/mqtt
 # github.com/mainflux/senml v1.0.0
 github.com/mainflux/senml
-# github.com/mattn/go-colorable v0.0.9
+# github.com/mattn/go-colorable v0.1.4
 github.com/mattn/go-colorable
-# github.com/mattn/go-isatty v0.0.3
+# github.com/mattn/go-isatty v0.0.11
 github.com/mattn/go-isatty
 # github.com/matttproud/golang_protobuf_extensions v1.0.1
 github.com/matttproud/golang_protobuf_extensions/pbutil
 # github.com/mitchellh/mapstructure v1.1.2
 github.com/mitchellh/mapstructure
-# github.com/nats-io/go-nats v1.6.0
-github.com/nats-io/go-nats
-github.com/nats-io/go-nats/encoders/builtin
-github.com/nats-io/go-nats/util
-# github.com/nats-io/nuid v1.0.0
+<<<<<<< HEAD
```

Comment:
Some bad merge?

Reply:
> Besides these remarks, in general looks like there is a huge number of vendor files pulled - this is suspicious. Please re-check.

Yea vendoring is huge, not sure actually how to check its your code :) 
What I did I just update with `go mod tidy` and `go mod vendor` to sync. Let me try again.

Reply:
@drasko There was a problem with vendoring yes, its fixed now.